userconfig.sh

```diff
@@ -124,7 +124,14 @@ import json
 
 def addEnv(dtext):
```

Comment:
This code exists here to be able to retrieve the user environment. If you no longer need to get it and write it in the kernel.json file, I would move it to the logic where we set the kernels (when we copy them from cvmfs). It would be more logical to do everything together there.